.devcontainer/Dockerfile

```diff
@@ -10,8 +10,7 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
 
 RUN apt install -y libgmp3-dev
 
-RUN rustup toolchain install nightly-2022-11-03 \
-    && rustup default nightly-2022-11-03 && rustup component add clippy && rustup component add rustfmt
+RUN rustup component add clippy && rustup component add rustfmt
```

Comment:
We should stick to `nightly-2023-05-28` since that is what we use in CI and elsewhere